addons/stock_account/report/report_stock_forecasted.py

```diff
@@ -12,9 +12,9 @@ def _compute_draft_quantity_count(self, product_template_ids, product_variant_id
         """ Overrides to computes the valuations of the stock. """
         res = super()._compute_draft_quantity_count(product_template_ids, product_variant_ids, wh_location_ids)
         domain = self._product_domain(product_template_ids, product_variant_ids)
-        svl = self.env['stock.valuation.layer'].search(domain)
-        currency = svl.currency_id or self.env.company.currency_id
-        value = float_repr(sum(svl.mapped('value')), precision_digits=currency.decimal_places)
+        quants = self.env['stock.quant'].search(domain + [('location_id', 'in', wh_location_ids)])
```

Comment:
I would keep the stock.valuation.layer and call convert to env.company_id.currency when the layer currency is different than the current one. (maybe with a read_group to earn time)

Reply:
As discussed elsewhere, the amount doesn't need to be converted since there can only be one currency displayed at the same time in this report.